4-WebApp-your-API/4-3-AnyOrg/Readme.md

```diff
@@ -205,6 +207,7 @@ Open the project in your IDE (like Visual Studio or Visual Studio Code) to confi
 1. Find the app key `TenantId` and replace the existing value with 'common'.
 1. Find the app key `ClientId` and replace the existing value with the application ID (clientId) of the `WebApi-MultiTenant-ToDoListService-v2` application copied from the Azure portal.
 1. Find the app key `ClientSecret` and replace the existing value with the key you saved during the creation of the `WebApi-MultiTenant-ToDoListService-v2` app, in the Azure portal.
+1. Find the app key `AllowedTenants` and add your tenant (for which you've registered the application). You can add as many tenants as you wish into this array and all of them will have access to Web API.
```

Comment:
>You can add as many tenants as you wish into this array and all of them will have access to Web API [](http://example.com/codeflow?start=104&length=99)

Add the tenant/Dirtectory Id that you noted earlier during registering tour app. you can also add more tenants as you wish into this array if you want apps from those tenant being able to access this Web API